src/functions/parseCommitBody.js

```diff
@@ -9,7 +9,7 @@ const parseCommitBody = function(body) {
     return [];
   }
 
-  const regEx = /((updates|resolves|closes)\s(\"[^\"]+\"|[^,\s]+))/gi;
+  const regEx = /((updates?|resolves?|closes?):? \"?([A-Z][A-Z0-9_]+\-[0-9]+)\"?)/gi;
```

Comment:
Changed regex to follow this site: https://confluence.atlassian.com/adminjiraserver072/changing-the-project-key-format-828787722.html